core/runner/src/main/java/com/cognifide/aet/runner/processing/processors/ProcessorStrategy.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.runner.processing.processors;
+
+import com.cognifide.aet.communication.api.messages.FinishedSuiteProcessingMessage;
+import com.cognifide.aet.communication.api.messages.FinishedSuiteProcessingMessage.Status;
+import com.cognifide.aet.communication.api.metadata.ValidatorException;
+import com.cognifide.aet.communication.api.wrappers.Run;
+import com.cognifide.aet.runner.RunnerConfiguration;
+import com.cognifide.aet.runner.processing.MessagesSender;
+import com.cognifide.aet.runner.processing.SuiteExecutionFactory;
+import com.cognifide.aet.runner.processing.SuiteProcessor;
+import com.cognifide.aet.runner.processing.data.RunIndexWrappers.RunIndexWrapper;
+import com.cognifide.aet.runner.processing.data.SuiteDataService;
+import com.cognifide.aet.vs.StorageException;
+import java.util.concurrent.Callable;
+import javax.jms.JMSException;
+import org.slf4j.Logger;
+import javax.jms.Destination;
+
+abstract public class ProcessorStrategy<T> implements Callable<String> {
```

Comment:
Please reorder the modifiers to comply with the Java Language Specification, the order should be `public abstract class`.